lib/rugged_adapter/git_layer_rugged.rb

```diff
@@ -388,11 +388,13 @@ def tree_entry(commit, path)
         tmp_entry = nil
 
         pathname.each_filename do |dir|
-          if tmp_entry.nil?
-            return nil unless (tmp_entry = commit.tree[dir])
-          else
-            tmp_entry = @repo.lookup(tmp_entry[:oid])[dir]
-          end
+          tmp_entry = if tmp_entry.nil?
```

Comment:
Let's make this a bit shorter:
```ruby
tmp_entry = tmp_entry ? @repo.lookup(tmp_entry[:oid])[dir] : commit.tree[dir]
return nil unless tmp_entry
```

Reply:
Thanks @bartkamphorst for the suggestion!